frog/imports/client/Wiki/helpers.js

```diff
@@ -105,12 +105,35 @@ const listWikis = async () => {
   });
 };
 
+// Function to get the pages of all pages in a wiki, returns [Title, ID]
```

Comment:
Would recommend Javadoc comments so that we are compatible with IDEs

Reply:
We should also specify this function return only valid pages.

Reply:
Made the respective changes